File: Ejercicios/C++/Ejercicio7.cpp
#include <iostream>
#include <vector>
using namespace std;
int main() {
    int arr[3] = {1, 2, 3}; // Estatico, no puede crecer
    vector<int> lista;      // Dinamico
    lista.push_back(1); lista.push_back(2);
    cout << "Size arreglo: 3 | Size vector: " << lista.size() << "\n";
    return 0;
}

File: Ejercicios/Java/Ejercicio7.java
import java.util.ArrayList;
public class Ejercicio7 {
    public static void main(String[] args) {
        int[] arr = new int[3];
        arr[0] = 1; arr[1] = 2; arr[2] = 3;
        System.out.println("Arreglo fijo length: " + arr.length);
        
        ArrayList<Integer> lista = new ArrayList<>();
        lista.add(1); lista.add(2);
        System.out.println("ArrayList dinamico size: " + lista.size());
    }
}
